src/Components/benchmarkapps/Wasm.Performance/dockerfile

```diff
@@ -29,8 +29,7 @@ RUN .dotnet/dotnet publish -c Release -r linux-x64 --sc true -o /app ./src/Compo
 RUN chmod +x /app/Wasm.Performance.Driver
 
 WORKDIR /app
-
-FROM mcr.microsoft.com/playwright/dotnet:v1.45.1-jammy-amd64 AS final
+FROM docker pull mcr.microsoft.com/playwright/dotnet:v1.46.0-noble AS final
```

Comment:
@MackinnonBuck you updated this like 5 months ago.

Is it fair that we need to grab the latest version from https://hub.docker.com/r/microsoft/playwright-dotnet?

Reply:
The version of the docker image needs to exactly match the version in [Versions.props](https://github.com/dotnet/aspnetcore/blob/02fbb9c82fa0eb7691b54d883d7a54524c3945ff/eng/Versions.props#L312). Otherwise, the benchmarks will fail to run. So if we make this change, we should also update the Playwright dependency for the rest of the repo.

Reply:
Thanks!

Is there a way this type of thing can be caught during the build?